WordPress/Classes/Utility/Migrations/87-88/BlogToBlogMigration87to88.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+class BlogToBlogMigration87to88: NSEntityMigrationPolicy {
```

Comment:
Should we be syncing settings for all blogs after the migration is completed? I was looking at the report card and I found surprising that the settings for each site weren't saved until I navigated to that site.

Reply:
On previous experiences, when we do a bunch of calls to this api, for some reason the setting is not stored properly on remote.

For example, from 5 blogs that we post to be set as `gutenberg`, one or two of them might be set to `aztec`. This happened on both iOS and Android.

That was the reason why we added the delay in the bulk post on previous iterations. That helped on reliability but didn't solve the problem completely.

So I'm thinking that is more reliable to lazily sync each site when it's needed.

Reply:
That makes me curious, but it's probably not worth the effort of fully investigating, the current approach seems like it should be fine